### PR TITLE
Add a document describing the change process

### PR DESCRIPTION
**Pull request by ramonza**

### CODE REVIEW

**Comment by ramonza** [comment]
This is my understanding of the change process we talked about offline.

Presented here for general comment.

/cc @dinooliva @bogdandrutu @rakyll @mtwo @shahprit

File: Process.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+At this point, we allow a week for anyone to comment on the issue to share their opinion.
+
+### Step 2: Approval
+
+New issues tagged with "proposal" that have been open at three business days will be discussed at the regular community meeting.
+If there is consensus, the issue can be approved directly. If more investigation is needed, a group of responsible
+individuals will be selected at the meeting to work on the issue.
+
+If the issue is approved, the "approved" should be added.
```
Comment (bogdandrutu) [inline Process.md]: I think you refer to label here.

**Comment by bogdandrutu** [comment]
This is great. Thanks @ramonza

File: Process.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+require Spec changes. These should be filed as issues on the individual OpenCensus library
+repository.
+
+## Process
+
+### Step 1: Opening an issue
+
+Open an issue on the (opencensus-specs repo)[https://github.com/census-instrumentation/opencensus-specs/issues/new].
+
+The issue should describe the problem you're trying to solve and the proposed solution (if any).
```
Comment (shahprit) [inline Process.md]: perhaps also call out the alternatives considered

File: Process.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+When opened, the label "proposal" should be applied. The issue will move out of "proposal" state when it is
```
Comment (shahprit) [inline Process.md]: Do you want to clarify the purpose of "proposal" label? should people not work on something unless it gets to approved state?

File: Process.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+approved in the next step.
+
+At this point, we allow a week for anyone to comment on the issue to share their opinion.
+
+### Step 2: Approval
+
+New issues labelled with "proposal" that have been open at three business days will be discussed at the regular community meeting.
```
Comment (shahprit) [inline Process.md]: I'm not sure if I follow this statement. what does "at three business days mean?

Comment (ramonza) [reply Process.md]: Ah should be "at least three"

File: Process.md (hunk 3 above)
Comment (ramonza) [reply Process.md]: Yes, will do

**Comment by bogdandrutu** [comment]
Please resync and submit this.

**Pull request merged**